packages/api/portfolioDrafts/getPortfolioDrafts.ts

```diff
@@ -37,6 +38,14 @@ export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayPr
   const params: ScanCommandInput = {
     TableName: TABLE_NAME,
     Limit: limit,
+    ProjectionExpression: Object.keys(portfolioDraftFields)
+      .join(", ")
+      .replace("name", "#name")
+      .replace("status", "#status"),
```

Comment:
This is one example of how to use part of it, to query the DB for expected fields